common/helpers.go

```diff
@@ -80,7 +77,7 @@ func convertArgument(
 		return convertBaseJSHandleTypes(ctx, execCtx, a)
 	default:
 		b, err := json.Marshal(a)
-		return &cdpruntime.CallArgument{Value: b}, err
+		return &cdpruntime.CallArgument{Value: b}, err //nolint:wrapcheck
```

Comment:
Added this to make the linter pass. I didn't fix it because it's not related to this PR.